packages/zowe-explorer/package.json

```diff
@@ -101,6 +101,18 @@
         "key": "ctrl+v",
         "mac": "cmd+v",
         "when": "focusedView == zowe.uss.explorer || viewItem =~ /^(directory.*|ussSession.*)/ && !listSupportsMultiselect"
+      },
+      {
+        "command": "zowe.ds.copyDataSet",
+        "key": "ctrl+c",
+        "mac": "cmd+c",
+        "when": "focusedView == zowe.ds.explorer"
+      },
+      {
+        "command": "zowe.ds.pasteMember",
```

Comment:
Should these commands be named consistently if they offer equivalent copy/paste behavior?
Either "copyDataSet" and "pasteDataSet", or "copyMember" and "pasteMember"

Reply:
Adressed in : a19a8de6821b088c5790d4b27484f48ff802a010